modules/setting/setting.go

```diff
@@ -985,6 +986,11 @@ func NewContext() {
 			log.Fatal(4, "Error saving generated JWT Secret to custom config: %v", err)
 		}
 	}
+
+	PasswordComplexity = sec.Key("PASSWORD_COMPLEXITY").MustString("[a-z]+[A-Z]+[0-9_]+[^A-Za-z0-9_]+")
+	if len(PasswordComplexity) == 0 {
```

Comment:
You don't need this again as you have passed the default value to `MustString`

Reply:
Good day!
I can redo the logic as follows:
if the setting is not specified or if there is not a single dictionary parameter, the check is completely disabled

So arrange?